src/types/types.tsx

```diff
@@ -0,0 +1,9 @@
+export interface IItem {
+  id: number;
+  text: string;
+  isDone: boolean;
+}
+
+export interface IIsDoneList {
+  isDoneList: boolean;
+}
```

Comment:
```suggestion
export interface IItem {
  id: number;
  text: string;
  isDone: boolean;
}

export type IItemList = IItem[]

export interface IIsDoneList {
  isDoneList: boolean;
}
```

`IItem[]` 형태가 많이 쓰이는데, 이러면 type alias를 쓰는게 좋겠죠